BenchmarkVCFs/ExportToGoogleSheets.ipynb

```diff
@@ -0,0 +1,315 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/broadinstitute/palantir-workflows/blob/mg_benchmark_compare/BenchmarkVCFs/ExportToGoogleSheets.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Export comparison data to Google Sheet\n",
+        "\n",
+        "This notebook will extract the comparison data generated with the `CompareBenchmarks` workflow and export it as a new sheet (tab) to a Google spreadsheet, including the required (conditional) formatting.\n",
+        "\n",
+        "__*Note*__: The Google authentication process requires this notebook to be run in [Google Colab](https://colab.research.google.com/) using the link above."
+      ],
+      "metadata": {
+        "id": "wuuQdfr022Nw"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Install the Firecloud API\n",
+        "\n",
+        "This is required in case the [Firecloud API](https://github.com/broadinstitute/fiss) isn't installed in the current runtime."
+      ],
+      "metadata": {
+        "id": "Zg0RWhvyXgQH"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "try:\n",
+        "    import firecloud.api as fapi\n",
+        "except ModuleNotFoundError:\n",
+        "    %pip install firecloud"
+      ],
+      "metadata": {
+        "id": "Wp4aXPPt4vtX"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Data export\n",
+        "\n",
+        "In the cell below, enter the `gs://` path to the `comparison.csv` file, which is the output of the `CompareBenchmarks` workflow.\n",
+        "\n",
+        "In addition, please enter the Google Spreadsheet ID of the notebook that you want to export the data to. The spreadsheet ID is shown in the URL in the browser. If you want to create a new spreadsheet you can do so on [sheets.google.com](https://sheets.google.com) or simply navigate to [sheet.new](https://sheet.new).\n",
```

Comment:
"Google Spreadsheet ID of the notebook" --> "Google Spreadsheet ID of the spreadsheet"